recipes/natcap-invest/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "natcap.invest" %}
+{% set version = "3.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/natcap/invest/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 0d9ce7ab5a9184c1cfcb2d2ec8295216e163b61e979b6bba3246342400945bdb
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - babel
+    - cython
+    - numpy >=1.11.0,!=1.16.0,<1.22.0
+  run:
+    - python
+    - babel
+    - chardet >=3.0.4   
+    - cython 
+    - gdal >=3.1.2,!=3.3.0
+    - flask
+    - numpy >=1.11.0,!=1.16.0,<1.22.0
+    - {{ pin_compatible('numpy') }}
+    - openpyxl
+    - pandas >=1.2.1  
+    - pint    
+    - psutil >=5.6.6
+    - pygeoprocessing >=2.3.2
+    - pyro4 ==4.77
+    - pyside2 !=5.15.0
+    - qtawesome
+    - qtpy >1.3
+    - requests
+    - rtree >=0.8.2,!=0.9.1
+    - scipy >=1.6.0
+    - shapely >=1.7.1,<2.0.0
+    - taskgraph[niced_processes] >=0.11.0
```

Comment:
`[]` have a different meaning in conda than in pip. Thus this won't work as expected.